llvm/lib/Target/AMDGPU/AMDGPUSwLowerLDS.cpp

```diff
@@ -0,0 +1,1335 @@
+//===-- AMDGPUSwLowerLDS.cpp -----------------------------------------===//
```

Comment:
The file seems to have the exec permission bits raised (typically happens when new files are created and merged on Windows).

Reply:
Fixed dc98482d9817a18242e35fb9947f81cc6021b63c